src/lib/holocene/input/input.stories.svelte

```diff
@@ -31,13 +32,13 @@
     value: { control: 'text' },
     name: { control: 'text' },
     icon: { control: 'select', options: iconNames },
-    theme: { control: 'radio', options: ['dark', 'light'] },
+    theme: { control: 'select', options: ['dark', 'light'] },
```

Comment:
Noting that this theme select doesn't appear to work and also there is a `dark` property being set in `args` above.